service_mesh/v2x/upgrading-ossm.adoc

```diff
@@ -386,16 +386,30 @@ spec:
     jaeger:
       name: jaeger
       install:
-        storage: Memory # or Elasticsearch
-        memory: {} # details specific to memory storage
-        elasticsearch: {} # details specific to elasticsearch storage
+        storage:
+          type: Memory # or Elasticsearch for production mode
+        memory:
+          maxTraces: 100000
+        elasticsearch: # the following values only apply if storage:type:=Elasticsearch
```

Comment:
`memory` and `elasticsearch` should come under `storage`

```
addons:
    jaeger:
      name: jaeger
      install:
        ingress:
          enabled: true
        storage:
          type: Memory # or Elasticsearch for production mode
          memory:
            maxTraces: 100000
          elasticsearch: # the following values only apply if storage:type:=Elasticsearch
            storage: # specific storageclass configuration for the Jaeger Elasticsearch (optional)
              size: "10G"
              storageClassName: "managed-premium"
            nodeCount: 3
            redundancyPolicy: SingleRedundancy
```

Reply:
Updated the indention levels to match.